css/styles.css

```diff
@@ -0,0 +1,142 @@
+* {
+	box-sizing: border-box;
+	font-family: Sahitya, sans-serif;
+	margin: 0px;
+}
+
+header {
+	background-color: black;
+	color: yellow;
+	height: 125px;
+	border-bottom: 5px solid yellow;
+	font-size: 20px;
+}
+
+h1 {
+	margin: 0px;
+	padding-left: 20px;
+}
+
+header a {
+	color: white;
+	font-size: 20px;
+	text-decoration: none;
+	padding-right: 15px;
+}
+
+.container {
+	width: 960px;
+	margin:0 auto;
+}
+
+.big {
+	font-size: 25px;
+	font-weight: bold;
+	text-transform: capitalize;
+}
+
+header div {
+	float: left;
+	padding-left: 150px;
+}
+
+h2 {
+	background-color: #a9a9a9;
+	text-align: center;
+	margin: 0px;
+	box-sizing: content-box;
+	font-size: 30px;
+	height: 70px;
```

Comment:
try to have the content define the height of a container, then apply the needed margin or padding.